src/api.js

```diff
@@ -0,0 +1,34 @@
+import axios from "axios";
+
+const ncNews = "https://oscar-northcoders-news.onrender.com";
```

Comment:
Check out our notes on axios for the axios recommended way to handle this baseURL